tests/e2e-tests/config/fixture-loaders.js

```diff
@@ -0,0 +1,297 @@
+/* eslint-disable no-console */
+/**
+ * External dependencies
+ */
+const WooCommerceRestApi = require( '@woocommerce/woocommerce-rest-api' )
+	.default;
+
+require( 'dotenv' ).config();
+
+const WooCommerce = new WooCommerceRestApi( {
+	url: `${ process.env.WORDPRESS_BASE_URL }:${ process.env.WORDPRESS_PORT }/`, // Your store URL
+	consumerKey: 'consumer_key', // Your consumer key
+	consumerSecret: 'consumer_secret', // Your consumer secret
+	version: 'wc/v3', // WooCommerce WP REST API version
+	axiosConfig: {
+		auth: {
+			username: process.env.WORDPRESS_LOGIN,
+			password: process.env.WORDPRESS_PASSWORD,
+		},
+	},
```

Comment:
consumer key and secret are not used currently (only placeholder to avoid errors) because we can't create them dynamically yet, for that we use Basic auth with user and password.